.github/workflows/linters.yml

```diff
@@ -0,0 +1,44 @@
+name: Linters
```

Comment:
- [Optional] Please make sure to use the present tense when you write the PR description